java/main/com/google/privacy/differentialprivacy/GaussianNoise.java

```diff
@@ -114,6 +114,32 @@ public MechanismType getMechanismType() {
     return MechanismType.GAUSSIAN;
   }
 
+  /**
+   * Returns {@link ConfidenceInterval} object with the given {@code confidenceLevel},
+   * using {@code noisedValue}, L_0 and L_inf sensitivities,
+   * and {@code epsilon} and {@code delta} for Gaussian distribution.
+   */
+  @Override
+  public ConfidenceInterval getConfidenceInterval(
+          double noisedValue, int l0Sensitivity, double lInfSensitivity,
+          double epsilon, Double delta, double confidenceLevel){
+    // TODO: Implement confidence interval computation.
+    return null;
+  }
+
+  /**
+   * Returns {@link ConfidenceInterval} object with integer bounds for the given {@code confidenceLevel},
```

Comment:
Similar to above, I would phrase this something like this:

 "Computes a confidence interval that contains the raw value integer {@code x} passed to {@link #addNoise(long, int, double, double, Double)} with a probability greater or equal to {@code confidenceLevel} based on the specified {@noisedValue} and noise parameters."